### PR TITLE
Update hybrid solver mocks and add a CQM data generator

### PR DESCRIPTION
**Pull request by randomir**

### CODE REVIEW

File: dwave/cloud/testing/mocks.py [hunk 1]
```diff
@@ -357,7 +352,34 @@ def hybrid_bqm_solver_data(**kwargs) -> dict:
...
+        description="Hybrid unstructured CQM mock solver",
+        supported_problem_types=["cqm"],
+        # solver-specific properties (mock values)
+        maximum_number_of_biases=2**20,
+        maximum_number_of_constraints=2**10,
+        maximum_number_of_linear_biases_real=2**20,
+        maximum_number_of_quadratic_variables=2**20,
+        maximum_number_of_quadratic_variables_real=0,
+        maximum_number_of_variables=2**16,
+        maximum_time_limit_hrs=1.0,
```
Comment (JoelPasvolsky) [inline dwave/cloud/testing/mocks.py:378]:
```suggestion
        maximum_time_limit_hrs=24.0,
```

File: dwave/cloud/testing/mocks.py [hunk 2]
```diff
@@ -357,7 +352,34 @@ def hybrid_bqm_solver_data(**kwargs) -> dict:
...
+    return unstructured_solver_data(**params)
+
+
+def hybrid_cqm_solver_data(**kwargs) -> dict:
+    params = dict(
+        id="hybrid_cqm_solver",
+        description="Hybrid unstructured CQM mock solver",
+        supported_problem_types=["cqm"],
+        # solver-specific properties (mock values)
+        maximum_number_of_biases=2**20,
```
Comment (JoelPasvolsky) [inline dwave/cloud/testing/mocks.py:372]: Why not 2e07 etc for these?

File: dwave/cloud/testing/mocks.py [hunk 3]
```diff
@@ -367,7 +389,13 @@ def hybrid_dqm_solver_data(**kwargs) -> dict:
     params = dict(
         id="hybrid_dqm_solver",
         description="Hybrid unstructured DQM mock solver",
-        supported_problem_types=["dqm"]
+        supported_problem_types=["dqm"],
+        # solver-specific properties (mock values)
+        maximum_number_of_biases=2**20,
+        maximum_number_of_cases=2**12,
+        maximum_number_of_variables=2**10,
+        maximum_time_limit_hrs=1.0,
```
Comment (JoelPasvolsky) [inline dwave/cloud/testing/mocks.py:397]:
```suggestion
        maximum_time_limit_hrs=24.0,
```

File: releasenotes/notes/add-asv-benchmarks-a28e767c704256b1.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    Add ``dwave.could.testing.mocks.hybrid_cqm_solver_data()``, a mock metadata
```
Comment (JoelPasvolsky) [inline releasenotes/notes/add-asv-benchmarks-a28e767c704256b1.yaml:4]:
```suggestion
    Add ``dwave.cloud.testing.mocks.hybrid_cqm_solver_data()``, a mock metadata
```

File: releasenotes/notes/add-asv-benchmarks-a28e767c704256b1.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+    generator for the CQM hybrid solver.
+fixes:
+  - |
+    Update ``dwave.could.testing.mocks.hybrid_*_solver_data()`` generators to
```
Comment (JoelPasvolsky) [inline releasenotes/notes/add-asv-benchmarks-a28e767c704256b1.yaml:8]:
```suggestion
    Update ``dwave.cloud.testing.mocks.hybrid_*_solver_data()`` generators to
```

**Review by JoelPasvolsky — APPROVED** [review]

**Comment by randomir** [comment]
These are just mock values, rounded down on purpose.

File: dwave/cloud/testing/mocks.py (hunk 2 above)
Comment (randomir) [reply dwave/cloud/testing/mocks.py:372]: Just adding dummy values (for now). I just wanted to capture the list of properties and order of magnitude for values..

**Pull request merged**